enzyme/BCLoad/BCLoader.cpp

```diff
@@ -6,6 +6,9 @@
 #include "llvm/Support/SourceMgr.h"
 #include "llvm/Support/raw_ostream.h"
 
+#include "BCLoader.h"
+#include "BCLoaderNew.h"
```

Comment:
Can you put these in a single header?

Reply:
Should I create another header or  move the new code into the old header?

Reply:
Into the old header is fine

Reply:
ok